README.md

```diff
@@ -258,124 +258,127 @@ var spotifyApi = new SpotifyWebApi();
 
 // Get multiple albums
 spotifyApi.getAlbums(['5U4W9E5WsYb2jUQWePT8Xm', '3KyVcddATClQKIdtaap4bV'])
-  .then(function(data) {
-    console.log('Albums information', data.body);
-  }, function(err) {
-    console.error(err);
-  });
+    .then(function (data) {
```

Comment:
Seems like you've got different style settings than the code in this repository, e.g. 4 spaces per tab, and space in `function (`. There's no right and wrong here, but in order to be consistent the style needs to stay the same as it is.